src/vmm/src/lib.rs

```diff
@@ -604,4 +603,29 @@ mod tests {
             if e.kind() == ErrorKind::InvalidInput
         ));
     }
+
+    #[test]
+    fn test_create_vcpus() {
+        // The scopes force the created vCPUs to unmap their kernel memory at the end.
```

Comment:
Unmapping the kernel memory is not needed since we are recreating the mock vmm.

Reply:
When we recreate it, the object isn't destroyed, it's shadowed (Rust lets us create a new object with the same name and doesn't let us access the old one anymore within the same scope, but *doesn't* immediately clean it up). I put a breakpoint in `KvmRunWrapper::drop` and it's never called until the end of the function, unless we explicitly enforce scopes. We can of course leave the memory there (I'm not sure if it `ENOMEM`s at the 256th vcpu) but I prefer the forced cleanup